src/Sulu/Bundle/SecurityBundle/Resources/config/doctrine/AccessControl.orm.xml

```diff
@@ -14,7 +14,7 @@
 
         <field name="permissions" type="smallint" column="permissions"/>
         <field name="entityId" type="integer" column="entityId"/>
-        <field name="entityClass" type="string" column="entityClass"/>
+        <field name="entityClass" type="string" column="entityClass" length="191"/>
```

Comment:
What should we do if the `entityClass` is longer than this?

Reply:
mysql would throw an exception (`Data too long for column 'entityClass'`). The current longest classname in my current project in the ro_routes table was 50 characters (`Sulu\Bundle\ArticleBundle\Document\ArticleDocument`).